flower-search/flow-index.yml

```diff
@@ -3,30 +3,30 @@ metas:
   prefetch: 10
 pods:
   loader:
-    yaml_path: yaml/craft-load.yml
-    replicas: $REPLICAS
+    uses: yaml/craft-load.yml
+    parallel: $PARALLEL
     read_only: true
   flipper:
-    yaml_path: yaml/craft-flip.yml
-    replicas: $REPLICAS
+    uses: yaml/craft-flip.yml
+    parallel: $PARALLEL
     read_only: true
   normalizer:
-    yaml_path: yaml/craft-normalize.yml
-    replicas: $REPLICAS
+    uses: yaml/craft-normalize.yml
+    parallel: $PARALLEL
     read_only: true
   encoder:
-    image: jinaai/hub.executors.encoders.image.torchvision-mobilenet_v2
```

Comment:
are we sure we do not prefer to be explicit about the model? So the user can see exactly what we are using?

Reply:
This is because when we build docker image for the flower example we want to use the local encoder